gcp/workers/worker/worker.py

```diff
@@ -498,6 +531,11 @@ def _do_update(self, source_repo, repo, vulnerability, relative_path,
 
     filter_unsupported_ecosystems(vulnerability)
 
+    # Generate Vanir signatures for test instance.
+    project = osv.utils.get_google_cloud_project()
+    if project and project in ('oss-vdb-test', 'test-osv'):
```

Comment:
nit:
```suggestion
    if project in ('oss-vdb-test', 'test-osv'):
```
evaluates to False if `project` is `''` (or falsy)